tests/nlu/classifiers/test_diet_classifier.py

```diff
@@ -503,3 +504,47 @@ async def test_train_persist_load_with_composite_entities(
     assert loaded.pipeline
     text = "I am looking for an italian restaurant"
     assert loaded.parse(text) == trained.parse(text)
+
+
+async def test_process_gives_diagnostic_data(
+    component_builder: ComponentBuilder, tmp_path: Path
+):
+    """Tests if processing a message returns attention weights as numpy array."""
+
+    _config = RasaNLUModelConfig(
+        {
+            "pipeline": [
+                {"name": "WhitespaceTokenizer"},
+                {"name": "CountVectorsFeaturizer"},
+                {"name": "DIETClassifier", RANDOM_SEED: 1, EPOCHS: 1},
+            ],
+            "language": "en",
+        }
+    )
+
+    (trainer, trained, persisted_path) = await train(
```

Comment:
can we use of one of the already trained interpreters? I'd like to avoid training another model as part of the tests as this makes the CI slow.

Reply:
🤔 How do you do that? All other tests that I can find also train the model.

Reply:
>  All other tests that I can find also train the model.

Training models without need in tests are the root cause why the CI is so slow. We have to be very considerate when and where we test models. 

> How do you do that?

You could e.g. use [this](https://github.com/RasaHQ/rasa/blob/046fd94a88dca3b7d166d0045c2c1775f3072c2e/tests/conftest.py#L100) fixture, load the model and get the `interpreter` from it.

Reply:
Ok, I'll try that. I'd then also update the other tests in this file.

Reply:
The other tests actually rely on specific model settings it seems. But I got it to work with my test.